### ISSUE TITLE
Using a generic field gives error Generic Error Type.ContainsGenericParameters is true

### ISSUE THREAD

**Issue by voslucas** [jburzynski/TypeGen#56]
TypeGen v2.2.1.
Using GenerationSpec because I don't want to use annotations.

The following generic class, with a generic _property_ **works**.

``` csharp
   class Test<T>
    {
        public T somename { get; set; }
    }
```
It is generated as:

``` typescript
export class Test<T> {
    somename: T;
}
```

However, as soon as i change the _property_ to a _field_, like this, i get an **error**:

``` csharp
   class Test<T>
    {
        public T somename;
    }
```
The error says: 

> GENERIC error : Cannot create an instance of SampleProject.Test`1[T] because Type.ContainsGenericParameters is true.

Stack trace:

>  at System.RuntimeType.CreateInstanceCheckThis()
>    at System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean wrapExceptions, Boolean skipCheckThis, Boolean fillCache)
>    at System.Activator.CreateInstance(Type type, Boolean nonPublic, Boolean wrapExceptions)
>    at TypeGen.Core.Generator.GetFieldValueText(FieldInfo fieldInfo) in D:\Google Drive\Projects\type-gen\src\TypeGen\TypeGen.Core\Generator.cs:line 450
>    at TypeGen.Core.Generator.GetClassPropertyText(MemberInfo memberInfo) in D:\Google Drive\Projects\type-gen\src\TypeGen\TypeGen.Core\Generator.cs:line 511
>    at TypeGen.Core.Generator.<GetClassPropertiesText>b__43_0(String current, MemberInfo memberInfo) in D:\Google Drive\Projects\type-gen\src\TypeGen\TypeGen.Core\Generator.cs:line 551
>    at System.Linq.Enumerable.Aggregate[TSource,TAccumulate](IEnumerable`1 source, TAccumulate seed, Func`3 func)
>    at TypeGen.Core.Generator.GetClassPropertiesText(Type type) in D:\Google Drive\Projects\type-gen\src\TypeGen\TypeGen.Core\Generator.cs:line 550
>    at TypeGen.Core.Generator.GenerateClassOrInterface(Type type, ExportTsClassAttribute classAttribute, ExportTsInterfaceAttribute interfaceAttribute) in D:\Google Drive\Projects\type-gen\src\TypeGen\TypeGen.Core\Generator.cs:line 390
>    at TypeGen.Core.Generator.Generate(Type type, Boolean initializeGeneration) in D:\Google Drive\Projects\type-gen\src\TypeGen\TypeGen.Core\Generator.cs:line 257
>    at TypeGen.Core.Generator.<Generate>b__24_0(IEnumerable`1 acc, KeyValuePair`2 kvp) in D:\Google Drive\Projects\type-gen\src\TypeGen\TypeGen.Core\Generator.cs:line 161
>    at System.Linq.Enumerable.Aggregate[TSource,TAccumulate](IEnumerable`1 source, TAccumulate seed, Func`3 func)
>    at TypeGen.Core.Generator.Generate(GenerationSpec generationSpec) in D:\Google Drive\Projects\type-gen\src\TypeGen\TypeGen.Core\Generator.cs:line 160
>    at TypeGen.Cli.Program.<>c__DisplayClass9_0.<Generate>b__4(IEnumerable`1 acc, GenerationSpec spec) in D:\Google Drive\Projects\type-gen\src\TypeGen\TypeGen.Cli\Program.cs:line 147
>    at System.Linq.Enumerable.Aggregate[TSource,TAccumulate](IEnumerable`1 source, TAccumulate seed, Func`3 func)
>    at TypeGen.Cli.Program.Generate(String projectFolder, String configPath) in D:\Google Drive\Projects\type-gen\src\TypeGen\TypeGen.Cli\Program.cs:line 143
>    at TypeGen.Cli.Program.Main(String[] args) in D:\Google Drive\Projects\type-gen\src\TypeGen\TypeGen.Cli\Program.cs:line 75
> 
>

**Comment by jburzynski** [comment]
Hi, sorry I didn't let you know on e-mail!

This should be resolved in TypeGen 2.3.0, released a couple of days ago.

**Comment by jburzynski** [comment]
closed due to inactivity
